music_album_store.rb

```diff
@@ -0,0 +1,91 @@
+require 'json'
+require_relative 'music_album'
+require_relative 'app'
+
```

Comment:
- [ ] Please remove this `require_relative 'app'` it is giving the user an error when he tries to use the app.
![image](https://user-images.githubusercontent.com/74744787/229319585-503387bd-f940-4d36-8281-0a530987db1d.png)